storage/src/main/java/tech/pegasys/teku/storage/server/rocksdb/RocksDbInstanceFactory.java

```diff
@@ -165,7 +170,7 @@ private static ColumnFamilyOptions createColumnFamilyOptions(
       final KvStoreConfiguration configuration, final Cache cache) {
     return new ColumnFamilyOptions()
         .setCompressionType(configuration.getCompressionType())
-        .setBottommostCompressionType(configuration.getBottomMostCompressionType())
```

Comment:
we should leave bottom most compression type in

Reply:
I'll let your PR merge that change since you've been testing that.

Reply:
added this back after our chat, don't know why I removed this